tests/actions/test_settings.py

```diff
@@ -0,0 +1,25 @@
+def test_debug_toolbar_settings_are_disabled_by_default(remove_from_module_cache):
+    with remove_from_module_cache("actions.settings"):
+        import actions.settings
+
+        assert actions.settings.DEBUG_TOOLBAR is False
+        assert "debug_toolbar" not in actions.settings.INSTALLED_APPS
+        assert (
+            "debug_toolbar.middleware.DebugToolbarMiddleware"
+            not in actions.settings.MIDDLEWARE
+        )
+
+
+def test_debug_toolbar_settings_are_enabled_when_environment_variable_set_to_true(
+    monkeypatch, remove_from_module_cache
+):
+    with remove_from_module_cache("actions.settings"):
+        monkeypatch.setenv("DJANGO_DEBUG_TOOLBAR", "1")
+        import actions.settings
+
+        assert actions.settings.DEBUG_TOOLBAR is True
+        assert "debug_toolbar" in actions.settings.INSTALLED_APPS
+        assert (
+            actions.settings.MIDDLEWARE[0]
+            == "debug_toolbar.middleware.DebugToolbarMiddleware"
+        )
```

Comment:
If this `assert` failed, then it would be really useful for the user to know why. The expression that follows the `assert` can be a tuple of the form `(condition, message)`.

```suggestion
        assert (
            actions.settings.MIDDLEWARE[0]
            == "debug_toolbar.middleware.DebugToolbarMiddleware"
        ), "Django Debug Toolbar's middleware must come as early as possible"
```

Notice that adding a message is preferable to adding a comment, because the user will see the message when they run the test; they won't have to locate the test and then read the comment.